akka-actor/src/main/java/akka/annotation/ApiMayChange.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Marks APIs that are meant to evolve towards becoming stable APIs, but are not stable APIs yet.
+ * 
+ * Evolving interfaces can change from one feature release to another release (i.e. 2.1 to 2.2) without up-front notice.
```

Comment:
> I think technically minors (2.4, 2.5) are actually major releases for akka

That's not true, see here: doc.akka.io/docs/akka/2.4/common/binary-compatibility-rules.html